lib.bevy.types.ts

```diff
@@ -0,0 +1,613 @@
+// generated with https://github.com/jakobhellermann/bevy_reflect_ts_type_export
```

Comment:
im guessing this will need to be updated with every bevy version bump?

Reply:
For now, but once Bevy 0.9 is released we can auto-generate the file from a `build.rs` script so there will be no manual effort required to sync.